test/api_image_tag_test.go

```diff
@@ -73,14 +73,29 @@ func (suite *APIImageTagSuite) TestImageTagFailToOverrideExistingPrimaryReferenc
 
 // TestImageTagFailToOverrideExistingTag tests fail.
 func (suite *APIImageTagSuite) TestImageTagFailToOverrideExistingTag(c *check.C) {
-	repo, tag := "registry.hub.docker.com/library/busybox", "1.25"
+	repo, tag := "localhost:5000/shouldnotoverridetag", "1.0"
 	tagRef := fmt.Sprintf("%s:%s", repo, tag)
-	command.PouchRun("pull", tagRef).Assert(c, icmd.Success)
-	defer DelImageForceOk(c, tagRef)
 
-	resp, err := suite.sendRequest(busyboxImage, repo, tag)
-	c.Assert(err, check.IsNil)
-	CheckRespStatus(c, resp, 400)
+	// create valid tag
+	{
+		command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)
+
+		resp, err := suite.sendRequest(busyboxImage, repo, tag)
+		c.Assert(err, check.IsNil)
+		CheckRespStatus(c, resp, 201)
+		defer DelImageForceOk(c, tagRef)
```

Comment:
why delete tagRef image？

Reply:
the `tagRef` image is created by the case. If we don't delete it, re-run this case will fail. It just is used to clean up.

Reply:
But the `busyboxImage` also will be remain, dont need to delele it？

Reply:
`busyboxImage` is created for the case suite and basically, all suites are related to this image. Therefore, we don't need to delete it.